x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshots.java

```diff
@@ -139,6 +142,12 @@ public SearchableSnapshots(final Settings settings) {
         this.settings = settings;
     }
 
+    public static void ensureValidLicense(XPackLicenseState licenseState) {
+        if (licenseState.isAllowedByLicense(License.OperationMode.ENTERPRISE) == false) {
```

Comment:
let's use platinum here to make it available in snapshot builds on cloud

Reply:
Sure, done in 45de644.